src/Compilers/VisualBasic/Portable/Symbols/Source/ImplementsHelper.vb

```diff
@@ -445,7 +445,7 @@ DoneWithErrorReporting:
                                             implementingProperty.GetPropertyKindText())
 
                 ElseIf ((implementedProperty.GetMethod Is Nothing) Xor (implementedProperty.SetMethod Is Nothing)) AndAlso
-                       implementingProperty.GetMethod IsNot Nothing AndAlso implementingProperty.SetMethod IsNot Nothing
+                       implementingProperty.GetMethod IsNot Nothing AndAlso implementingProperty.SetMethod IsNot Nothing Then
```

Comment:
> Then [](start = 120, length = 5)

Consider reverting this file to its original state as this is the only change in it, which could be submitted in a separate PR.